### ISSUE TITLE
we will need a way to specify RNA identifiers

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1505]
using RNA central, for non-protein coding entities
https://twitter.com/fungidb/status/1007628995066761218

**Comment by kimrutherford** [comment]
Currently a Canto instance has to get all its genes from one source.  The current options are UniProt, Chado or Canto's own database.  Adding a new source isn't too tricky but there is some work involved in supporting more the one source at the same time.

**Comment by ValWood** [comment]
Yup,  there aren't so many RNA papers yet so this can happen later. It hasn't arisen previously because for pombe we just configure with our gene list, and I didn't think about it until I saw this paper.

**Comment by ValWood** [comment]
It might be worth punting this until the more  important tasks are done since there are not many examples yet.

@CuzickA 
@jseager7
